src/main/java/com/github/fge/jsonpatch/JsonPatchOperation.java

```diff
@@ -94,6 +94,14 @@ protected JsonPatchOperation(final String op, final JsonPointer path)
     public abstract JsonNode apply(final JsonNode node)
         throws JsonPatchException;
 
+    public String getOp() {
+        return op;
```

Comment:
Should this protectively copy the string?

Reply:
I wouldn't have thought this is necessary as `String` is immutable, if I'm not thinking of something can you please explain?

Reply:
Ah, you're right.